finagle-core/src/test/scala/com/twitter/finagle/loadbalancer/LoadBalancerFactoryTest.scala

```diff
@@ -40,14 +40,14 @@ class LoadBalancerFactoryTest extends AnyFunSuite with Eventually with Integrati
       perHostStats.let(true) {
         client
           .configured(LoadBalancerFactory.HostStats(sr))
-          .newService(port)
+          .newService(port)("test")
```

Comment:
question: If we keep the perHostFlag as App, we should not need to initiate requests here and below, why do you prefer this way?

Reply:
Maybe I launch test wrong, but on exactly failed test result always successful. When I made this changes, test has showing correct result.